Formula/mongodb-community@4.2.rb

```diff
@@ -0,0 +1,81 @@
+class MongodbCommunityAt42 < Formula
+  desc "High-performance, schema-free, document-oriented database"
+  homepage "https://www.mongodb.com/"
+
+  # frozen_string_literal: true
+
+  url "https://fastdl.mongodb.org/osx/mongodb-macos-x86_64-4.2.8.tgz"
+  sha256 "0ae9089e00c70c5f7b73dd267426f0df6a03fe2ba7f27729ee1341240cef7569"
+
+  bottle :unneeded
+
+  keg_only :versioned_formula
+
+  def install
+    prefix.install Dir["*"]
+  end
+
+  def post_install
+    (var/"mongodb").mkpath
+    (var/"log/mongodb").mkpath
+    if !(File.exist?((etc/"mongod.conf"))) then
+      (etc/"mongod.conf").write mongodb_conf
+    end
+  end
+
+  def mongodb_conf; <<~EOS
+    systemLog:
+      destination: file
+      path: #{var}/log/mongodb/mongo.log
+      logAppend: true
+    storage:
+      dbPath: #{var}/mongodb
+    net:
+      bindIp: 127.0.0.1
```

Comment:
Since this is a default config that only gets applied when there is no existing config, would it make sense to have auth enabled by default here?
```
security:
  authorization: "enabled"
```

Reply:
Sadly, there seems to be no available emoji for "can of worms".

Reply:
Yeah, that's why I asked...